package.json

```diff
@@ -15,23 +15,28 @@
   "homepage": "https://localhost:6060/",
   "scripts": {
```

Comment:
1. Please add  the following:
  a. Contributors: https://github.com/mojaloop/simulator/blob/master/package.json#L8
  b. Engines: https://github.com/mojaloop/simulator/blob/master/package.json#L20
2. Why is the license MIT? I assume this should be apache2.0? https://github.com/mojaloop/simulator/blob/master/package.json#L7